docs/nightly/en/release-notes/all-releases.md

```diff
@@ -1,5 +1,9 @@
 # All releases
 
+## 0.8
+
+* [0.8.0](/release-notes/release-0-7-2.md): 2024-05-18
```

Comment:
The link is wrong, it's v0.7.2. 😢  @WenyXu

Reply:
A fix - https://github.com/GreptimeTeam/docs/pull/971